### PR TITLE
added blog Top benefits of using online form builder

### PR DESCRIPTION
**Pull request by anirudh-tantry**
added blog Top benefits of using online form builder

### CODE REVIEW

File: content/blog/top-benefits-of-using-online-form-builder.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+- Also, make sure you understand how the form builder works. Some require you to upload files, while others don't.
+
+- Finally, make sure you test your form thoroughly. It may seem obvious, but testing your form is important because it helps you identify problems before they become major issues.
+
+In the Formester blog, we have already covered this topic — ["Best Online Form Builders 2022: How to Choose the Ideal One?"](https://formester.com/blog/best-online-form-builders/). We have listed out the factors to consider before you choose which tool will be the right choice for you.
+
+## Why should you opt Formester
+
+![Formester website](/blog/top-benefits-of-using-online-form-builder/top-benefits-of-using-online-form-builder.png 'Formester website')
```
Comment (piyushsinghania) [inline content/blog/top-benefits-of-using-online-form-builder.md:75]: 1. Should have a better title
2. The image name is incorrect, it should be name accordingly

File: content/blog/top-benefits-of-using-online-form-builder.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+While there are plenty of options available for users to select any of the top listed online form builders, here are some key reasons you should consider using Formester to build your first ever form for your business.
+
+### 1. You get full control
+
+Each online form builder that’s available has its own pros and cons. But one of the major benefits of using Formester is that it helps you have control of the form building interface with an extremely easy drag-and-drop option and is intuitive in nature.
+
+### 2. Easy to build
+
+The other major pluses of using Formester is its ease to build drag-and-drop features that require little to no instructions. The [text-easy HTML form builder](https://formester.com/) dashboard is designed in such a way that a user can explore the different tools that’s available without feeling overwhelmed.
```
[the remark this answers is not in the document]
Comment (piyushsinghania) [reply content/blog/top-benefits-of-using-online-form-builder.md:85]: Attaching an example here (for what do I mean by title) - 

image

File: content/blog/top-benefits-of-using-online-form-builder.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (piyushsinghania) [reply content/blog/top-benefits-of-using-online-form-builder.md:75]: There should be bit more room - 

image

Comment (ankitsinghaniyaz) [reply content/blog/top-benefits-of-using-online-form-builder.md:75]: Screenshot showing drag and drop builder from Formester

File: content/blog/top-benefits-of-using-online-form-builder.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ankitsinghaniyaz) [reply content/blog/top-benefits-of-using-online-form-builder.md:85]: Do we have the blog if not it's okay for now. Let's ask Vivek if he can give this.

File: content/blog/top-benefits-of-using-online-form-builder.md (hunk 1 above)
Comment (anirudh-tantry) [reply content/blog/top-benefits-of-using-online-form-builder.md:75]: What title image name should I give?

**Review by piyushsinghania — APPROVED** [review]

**Pull request merged**